rbloom.pyi

```diff
@@ -24,10 +24,10 @@ class Bloom:
 
     # load from file, see section "Persistence"
     @classmethod
-    def load(cls, filepath: str, hash_func: Callable[[Any], int]) -> Bloom: ...
+    def load(cls, dest: Union[bytes, str, BinaryIO], hash_func: Callable[[Any], int]) -> Bloom: ...
 
-    # save to file, see section "Persistence"
-    def save(self, filepath: str) -> None: ...
+    # save to file, file object, or return bytes, see section "Persistence"
+    def save(self, source: Optional[Union[str, BinaryIO]] = None) -> Union[bytes, None]: ...
```

Comment:
You can type the stub more accurately with the `@overload` decorator:

```python
import typing.overload

@overload
def save(self, dest: Union[str, BinaryIO]) -> None: ...

@overload
def save(self, dest: None = None) -> bytes: ...
```

Reply:
Also, when we can update to pyo3 0.20, using `PathBuf` will mean we also can take [`os.PathLike`](https://docs.python.org/3/library/os.html#os.PathLike) (https://github.com/PyO3/pyo3/pull/3374)